types/undertaker/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "async-done": "^1.3.2"
```

Comment:
There's a slight danger here that `async-done` could change it's `AsyncTask` return type in a future minor version and break this definition unexpected if we depend on version `^###`. I'd prefer if this was changed to `~1.3.2`.
In my experience, broken TS type definitions seem to happen more frequently that JS breaking changes and packages sometimes don't follow semversion rules when it comes to TS definition files.

Reply:
Ok 6615df0d47fea318d911c96af0a48d2e64c8a3f1